Gamma_Analysis.py

```diff
@@ -176,6 +176,15 @@ def peak_measurement(M, energy, sub_regions='both'):
 
     region_size = 1.3
     compton_distance = 4
+
+    # Cs134 compton region using Bi214 609 peak.
+    bi_fwhm = 0.05 * (609.31)**0.5
```

Comment:
Careful, the region will need to be the same number of channels as the other ROIs. So instead of using the 609 keV FWHM, use the FWHM of the given peak energy.

(It actually makes a difference in the USS Independence background that I was testing on.)